modules/client/src/controllers/ConditionalTransferController.ts

```diff
@@ -65,7 +65,7 @@ export class ConditionalTransferController extends AbstractController {
     const ret = await this.handleLinkedTransfers(params);
 
     // set recipient and encrypted pre-image on linked transfer
-    const recipientPublicKey = fromExtendedKey(recipient).publicKey;
+    const recipientPublicKey = fromExtendedKey(recipient).derivePath("0").publicKey;
```

Comment:
can we replace this with `xpubToAddress` fn (and any other time we do this)

Reply:
Tbh, `xpubToAddress` is a kind of confusing name. Maybe it should be `xpubToZeroAddress` or something.

Reply:
Realizing we can't actually replace this because we need the public key not the ETH address.